### PR TITLE
12.0 mig mail_attach_existing_attachment

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

File: mail_attach_existing_attachment/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+To use this module, you need to:
+
+* Adding some attachments on an object
+
+.. figure:: static/description/attachment.png
```
Comment (chienandalu) [inline mail_attach_existing_attachment/readme/USAGE.rst]: This isn't valid anymore, isn't it?

**Review by chienandalu — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
@ernestotejeda please squash directly all migration commits as there are no reviews yet

File: mail_attach_existing_attachment/readme/USAGE.rst (hunk 1 above)
Comment (ernestotejeda) [reply mail_attach_existing_attachment/readme/USAGE.rst]: image path was incorrect, i already have fixed

File: mail_attach_existing_attachment/readme/USAGE.rst [hunk 2]
```diff
@@ -2,10 +2,10 @@ To use this module, you need to:
 
 * Adding some attachments on an object
 
-.. figure:: static/description/attachment.png
+.. figure:: mail_attach_existing_attachment/static/description/attachment.png
```
Comment (chienandalu) [inline mail_attach_existing_attachment/readme/USAGE.rst:5]: I mean Odoo removed the option from the UI: https://github.com/odoo/odoo/issues/27760 So this instruction isn't valid

**Review by chienandalu — APPROVED** [review]
Great! :+1:

**Review by pedrobaeza — APPROVED** [review]

File: mail_attach_existing_attachment/readme/USAGE.rst (hunk 2 above)
Comment (sswapnesh) [reply mail_attach_existing_attachment/readme/USAGE.rst:5]: @chienandalu I don't know the module functionality but https://github.com/odoo/odoo/pull/30636 adds few options related to this

**Pull request merged**